examples/analog_hamiltonian_simulation/01_Introduction_to_Aquila.ipynb

```diff
@@ -7,175 +7,205 @@
     "tags": []
```

Comment:
<p>1) "average Rydberg density equal to 1/4" Why 1/4. There are three atoms, I'd expect to see 1/3 average.</p><p> </p><p>2) "Here the second term is the interaction term which is fixed once the setup is defined" Let's add a bit more details to this: i) van der Waals interactio increases quickly with decreasing distance. ii) We will use an Omega_max = X, which gives us a blockade radius of Y."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>It was 1/4 because i picked Omega=Delta, and the specific duration chosen. But I have changed it to Delta=0, such that the expected density is 1/3. The blockade radius will be mentioned below (not in this cell)</p>